doc/classes/@GlobalScope.xml

```diff
@@ -176,6 +176,17 @@
 				Returns the point at the given [param t] on a one-dimensional [url=https://en.wikipedia.org/wiki/B%C3%A9zier_curve]Bézier curve[/url] defined by the given [param control_1], [param control_2], and [param end] points.
 			</description>
 		</method>
+		<method name="blend_from_angle">
+			<return type="float" />
+			<param index="0" name="init" type="float" />
```

Comment:
I heavily recommend renaming this argument. `init` has a very specific connotation in programming. The surrounding functions in the global scope call this argument `start`

Reply:
Disagree. In interpolation, the term start can cause confusion. It is an angle of reference, not an angle that would be the same if the weight were 0.

Reply:
So be it then. `rest` sounds better, although it is very specific to bones.